README.md

```diff
@@ -165,12 +165,9 @@ container to the docker group for socket permissions.
 
 ### `token`
 
-[Generate a Personal Access Token (classic)](https://github.com/settings/tokens), with the `repo:public_repo` scope for only public repositories or the `repo` scope for public and private repositories, and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
+[Generate a GitHub Personal Access Token (fine-grained is recommended)](https://github.com/settings/tokens) (see the [GitHub authentication docs](https://docs.renovatebot.com/modules/platform/github/#authentication)) and add it to _Secrets_ (repository settings) as `RENOVATE_TOKEN`.
 You can also create a token without a specific scope, which gives read-only access to public repositories, for testing.
-This token is only used by Renovate, see the [token configuration](https://docs.renovatebot.com/self-hosted-configuration/#token), and gives it access to the repositories.
```

Comment:
@viceice let me know if this line is really useful, AFAIK it doesn't really explain anything 🤷